certify-core/src/main/java/io/mosip/certify/core/dto/CredentialMetadata.java

```diff
@@ -17,7 +18,9 @@ public class CredentialMetadata {
     private String id;
     private String format;
     private String scope;
-    private List<String> proof_types_supported;
+    // proof_types_supported is a map from v13 & an array before
+    private Object proof_types_supported;
     private List<String> types;
+    private Map<String, String> background_image;
```

Comment:
Again, is someone using this ?

Reply:
Right now, I'm passing in a blank link, should I remove it from the .properties example?

Reply:
Not being used, hence removing